kof-operator/internal/k8s/secrets.go

```diff
@@ -0,0 +1,35 @@
+package k8s
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetKubeconfigSecrets(ctx context.Context, k8sClient client.Client) (*corev1.SecretList, error) {
```

Comment:
We have a similar logic in the [remote_secret_manager](https://github.com/k0rdent/kof/blob/main/kof-operator/internal/controller/istio/remote-secret/remote_secret_manager.go#L88), need to remove the duplication